gnome/virt-manager/Portfile

```diff
@@ -25,7 +26,10 @@ long_description \
     allocation & virtual hardware. An embedded VNC and SPICE client \
     viewer presents a full graphical console to the guest domain. \
     \
-    The primary use on macOS is for remote administration of Linux boxes.
+    The primary use on macOS is for remote administration of Linux boxes. \
+    \
+    IMPORTANT: This application requires an X11 server which is not \
+    automatically installed. See for example https://www.xquartz.org .
```

Comment:
Could you please elaborate on why this is the case? What error do you get without one present? I haven't used this port in a while, but when I did, I installed the Quartz variant of the Gnome libraries. I don't think we can phrase X11 as a _requirement_…

(Also, if we do want this message, we should put it in a `note` rather than the description.)

Reply:
Please see [this mailing list thread](https://lists.macports.org/pipermail/macports-users/2020-May/048465.html). I have lost quite some time with this issue, suspecting errors in my SSH configuration. Currently "port info" makes no mention of having to manually install an X11 server, and given that libvirt supports VNC and SPICE it is not self-evident that X11 is required at all.

Reply:
Moved additional text from `long_description` to `notes`.